src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -90,10 +118,27 @@ public int countImages() {
     return images.size();
   }
 
+  public int countHeaders() {
+    return headers.size();
+  }
+
   public String getBackgroundColor() {
     return component.get(0).getCssValue("background-color");
   }
 
+  public String getBorderStyle() {
+    JavascriptExecutor js = (JavascriptExecutor)driver;
+    WebElement selectedComponent = component.get(0);
+    wait.forElementVisible(selectedComponent);
+    selectedComponent.click();
+
+    String script = "return window.getComputedStyle("
+                    + "document.querySelector('.active'),':before').getPropertyValue('Border')";
+    String borderValues = js.executeScript(script).toString();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "borderValues". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1488)